back/engines/commercial/bulk_import_ideas/spec/services/side_fx_bulk_import_service_spec.rb

```diff
@@ -19,6 +20,11 @@
           ideas.last.created_at,
           { payload: { model: 'idea', format: 'xlsx', items_created: 3, users_created: 2 } }
         )
+
+      expect { service.after_success(current_user, phase.id, 'idea', 'xlsx', ideas, users) }
```

Comment:
@kogre I don't fully get why this test fails.. Any advice?

Reply:
Feel free to merge it with the faillure, I can look into it later or next week, in the worst case